src/Particle.js

```diff
@@ -2525,6 +2549,17 @@ class Particle {
 		return product ? `/v1/products/${product}/devices/${deviceId}` : `/v1/devices/${deviceId}`;
 	}
 
+	/**
+	 * Helper for building subspace API paths based on org being provided or not
+	 * @param {string | undefined} org slug or ID
+	 * @param {string} path
+	 * @returns {string} path
+	 * @private
+	 */
+	_subspacePath(org, path) {
```

Comment:
subspace is an internal name. How about `namespacedPath` for this helper? It's going to be generally useful in other contexts too.

Reply:
Good call. Updating now